WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java

```diff
@@ -226,9 +227,9 @@ protected void finalize() throws Throwable {
 
     public void saveSettings() {
         SiteSettingsTable.saveSettings(mSettings);
-        siteSettingsPreferences(mActivity).edit().putString(LANGUAGE_PREF_KEY, mSettings.language).apply();
-        siteSettingsPreferences(mActivity).edit().putInt(DEF_CATEGORY_PREF_KEY, mSettings.defaultCategory).apply();
-        siteSettingsPreferences(mActivity).edit().putString(DEF_FORMAT_PREF_KEY, mSettings.defaultPostFormat).apply();
+        siteSettingsPreferences(mContext).edit().putString(LANGUAGE_PREF_KEY, mSettings.language).apply();
```

Comment:
I know this is pre-existing code, but how about using the builder pattern here?

```
siteSettingsPreferences(mContext).edit()
        .putString(LANGUAGE_PREF_KEY, mSettings.language)
        .putInt(DEF_CATEGORY_PREF_KEY, mSettings.defaultCategory)
        .putString(DEF_FORMAT_PREF_KEY, mSettings.defaultPostFormat)
        .apply();
```

Reply:
Makes sense, done in c97620d.